data/provider/src/main/java/tech/pegasys/teku/api/ChainDataProvider.java

```diff
@@ -618,18 +625,21 @@ private Map<Integer, Integer> getCommitteeIndices(
     for (BLSPublicKey key : committeeKeys) {
       final Optional<Integer> validatorIndex = spec.getValidatorIndex(state, key);
       if (validatorIndex.isEmpty()) {
+        i++;
         continue;
       }
       if (validators.contains(key.toHexString())
           || validators.contains(validatorIndex.toString())) {
-        output.put(i++, validatorIndex.get());
+        output.put(i, validatorIndex.get());
```

Comment:
can simplify here maybe...
`if (validatorIndex.isPresent() && (validators.contains || validators.contains))`
probably also make sure the spec.getValidatorIndex handles silly values...
by doing the 1 if statement, the i++ is at the bottom, no need for `continue`